### PR TITLE
Create first title and titles.scss file.

### PR DESCRIPTION
**Pull request by marcalmon**
## Summary

- Completed ths page titles.

## Screenshots
Screenshot from 2019-06-10 12-50-45

## Trello Card

https://trello.com/c/h8onT0E9/21-page-titles

### CODE REVIEW

File: scss/titles.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.title {
+	color: #000;
```
Comment (pabloccid) [inline scss/titles.scss]: You can move both this color and the size to constants, so they can be reused in the whole app 😁 
In order to do this, you may create colors.scss and sizes.scss files, that should contain only constants.

Comment (gabrielizalo) [reply scss/titles.scss]: @marcalmon recuerda en los archivos que te menciona Pablo agregar los valores que te menciona esta card: https://trello.com/c/BRwhE9Qo/11-setup-del-proyecto

File: scss/sizes.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+$nav-height: 70px;
```
Comment (pabloccid) [inline scss/sizes.scss]: You have many sizes that arent being used yet.

File: scss/colors.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+$background-card: #BED23C;
```
Comment (pabloccid) [inline scss/colors.scss]: You should use a name that describes the color. For example, `green`. You dont know where are you going to use this color, so it shouldnt reference its use

File: scss/colors.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+$blue: #00ADEE;
+$color-title: #000;
```
Comment (pabloccid) [inline scss/colors.scss]: Same case. `black`

File: scss/sizes.scss [hunk 5]
```diff
@@ -0,0 +1,6 @@
+$size-title: 35px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+$title-decoration-height: 90px;
```
Comment (pabloccid) [inline scss/sizes.scss]: This and the next one arent being used neither

**Pull request merged**